projects/sbb-esta/angular-business/src/lib/contextmenu/contextmenu/contextmenu.component.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  AfterContentInit,
+  Component,
+  ContentChild,
+  OnInit,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+import { DropdownComponent, DropdownTriggerDirective } from '@sbb-esta/angular-public';
+
+@Component({
+  selector: 'sbb-contextmenu',
+  templateUrl: './contextmenu.component.html',
+  styleUrls: ['./contextmenu.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class ContextmenuComponent implements OnInit, AfterContentInit {
+  /**
+   * Reference to a dropdown instance.
+   */
+  @ContentChild(DropdownComponent, { static: true }) dropdown: DropdownComponent;
+  /**
+   * Reference to a dropdown trigger directive instance.
+   */
+  @ViewChild(DropdownTriggerDirective, { static: true }) sbbDropdownDirective;
+
+  constructor() {}
+
+  ngOnInit(): void {}
+
+  ngAfterContentInit(): void {
+    /**
+     * Set the panel width
+     */
+    this.dropdown.panelWidth = '227px';
```

Comment:
Is the fixed width intended? What is the behavior if this is not limited?

Reply:
The fixed width is intended because the dropdown takes the width of the parent component to define its size. In this case the parent component will be the button which wraps the icon resulting in a panel with 24px width.